app/main.py

```diff
@@ -1,13 +1,31 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "line" : error["line_number"],
+        "column" : error["column_number"],
+        "message" : error["text"],
+        "name" : error["code"],
+        "source" : "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "errors":
+        [format_linter_error(separate_dict) for separate_dict in errors],
+        "path": file_path ,
+        "status": "failed"
```

Comment:
The 'status' is hardcoded to 'failed'. Consider checking if 'errors' is empty and set 'status' to 'passed' if there are no errors, similar to the logic used in the 'format_linter_report' function.